randovania/games/fusion/gui/ui_files/games_tab_fusion_widget.ui

```diff
@@ -136,17 +136,58 @@
+           <string>Randovania and the MARS patcher make some changes to the original game in order to improve the game experience or to simply fix bugs in the original game. Key differences are: 
+
+### Gameplay
+
+- The vanilla event system has been removed and replaced with non-linear event/flag handling.
+
+- Beams and Missile upgrades have been split. More details found on the FAQ tab.
+
+- Screw Attack will no longer allow Single Wall Jumps. Wall Jump behaviour is now the same as without Screw Attack. 
+
+- Red X drop chances have been slightly increased. Red X drops are now guarunteed when both Health and Missiles are full to help replenish Power Bombs.
+
+- Hints have been placed at Navigation Stations. More details found on the Hints tab.
+
+- In order to fight the SA-X, you will need to collect a configurable amount of infant Metroids, a way to approach the door to the Operations Room, Charge Beam and Missiles. Plasma Beam is not required to defeat the SA-X, and the Level 4 Keycard is not required to go to the Operations Room.
+
+- Changing course in Operations Room no longer locks any hatches.
+
+- The escape timer only triggers when entering Main Deck Docking Bay Hangar to fight the Omega Metroid.
```

Comment:
I would rephrase this as "During the escape hatches will no longer lock, and the timer will only trigger when entering ..."